native/build.gradle

```
@@ -70,8 +70,7 @@ tasks.withType(Checkstyle) {
 }
 
 spotbugsMain.finalizedBy validateSpotbugs
-checkstyleMain.dependsOn(":build-config:checkstyle:downloadMultipleFiles")
-
+checkstyleMain.dependsOn(":checkstyle:downloadMultipleFiles")
```

Shall we change this task to `downloadCheckstyleRuleFiles`? 

```suggestion
checkstyleMain.dependsOn(":checkstyle:downloadMultipleFiles")
```

---

Updated in 73efe20